gunicorn.conf.py

```diff
@@ -0,0 +1,22 @@
+# pylint: disable=invalid-name
+import gunicorn
+
+# Tell gunicorn to run our app
+wsgi_app = "cms.wsgi:application"
+
+# Replace gunicorn's 'Server' HTTP header to avoid leaking info to attackers
+gunicorn.SERVER = ""
+
+# Restart gunicorn worker processes every 1200-1250 requests
+max_requests = 1200
+max_requests_jitter = 50
+
+# Log to stdout
+accesslog = "-"
+
+# Time out after 25 seconds (notably shorter than Heroku's)
+timeout = 25
```

Comment:
Can we make this an env var and default to 25. We historically ran with 0 when running in GCP as Google recommends to let LB handle this. I'm not sure what it will be in AWS yet.